pkg/storage/driver/cfgmaps.go

```diff
@@ -107,7 +107,7 @@ func (cfgmaps *ConfigMaps) List(filter func(*rspb.Release) bool) ([]*rspb.Releas
 			continue
 		}
 
-		rls.Labels = filterSystemLabels(item.ObjectMeta.Labels)
+		rls.Labels = item.ObjectMeta.Labels
```

Comment:
should the same change be made for `func (cfgmaps *ConfigMaps) Query(labels map[string]string) ([]*rspb.Release, error) {` ?

Reply:
When I started this PR, I didn't make this change, cause `Query` is used only for internal purposes as far as I understood (used during upgrade, delete and etc). So it doesn't affect user, cause not involved in helm list.

Right now I checked all code places where `filterSystemLabels()` is used and dropped it if it doesn't brake unit tests.
So far there're only two use cases where this function is used:
- Get operation -  in Get operation tests, expect the release after creation to be deepEquals. Potentially it's possible do drop it from there, but I don't know which side effects it can cause.
- SQL driver - in SQL driver system labels are stored separately in corresponding table columns, so we really need such filtering in SQL driver code.

Reply:
Query is part of the public API and could be used by downstream users. Will the change, or lack thereof, cause confusion if this public function is used?